data/sql/updates/pending_db_world/rev_1682394052134857558.sql

```diff
@@ -0,0 +1,42 @@
+
+UPDATE `creature_template` SET `AIName`= 'SmartAI',`ScriptName`='' WHERE `entry` IN (18445,18369,20812);
+UPDATE `gameobject_template` SET `Data2`=300000, `AIName`='SmartGameObjectAI', `ScriptName`='' WHERE `entry` IN (182521,182349,182350);
```

Comment:
Just some formatting here

```suggestion
UPDATE `creature_template` SET `AIName`= 'SmartAI', `ScriptName`='' WHERE `entry` IN (18445,18369,20812);
UPDATE `gameobject_template` SET `Data2`=300000, `AIName`='SmartGameObjectAI', `ScriptName`='' WHERE `entry` IN (182521,182349,182350);
```